frontend/src/shared/components/SelectorCard.js

```diff
@@ -0,0 +1,144 @@
+/*
+Externally Passed:
+- bool :        Selected // Set to true or false to control button display and handling
+- handler :     HandleSelect // Occurs once the "select" button has been pushed
+- handler :     HandleUnselect // Occurs once the "unselect" button is pushed
+ */
+
+import React, {Component, Fragment} from 'react'
+import {Button, Row, Col, Container, CardBlock, Collapse} from "reactstrap";
+import PropTypes from 'prop-types'
+import styled from "styled-components";
+
+const Card = styled.div`
+  background-color: #f8f9fa;
+  border-radius: 5px;
+  margin-top: 15px;
+  padding: 10px;
+  width: 100%;
+`
+
+class SelectorCard extends Component {
+  static propTypes = {
+    title: PropTypes.object,
+    body: PropTypes.object,
+    dataPoint: PropTypes.object.isRequired,
+    selected: PropTypes.bool.isRequired,
+    preExpand: PropTypes.func,
+    onToggle: PropTypes.func,
+    onEdit: PropTypes.func,
+    onDelete: PropTypes.func,
+  };
+
+  static defaultProps = {
+    title: "",
+    body: "",
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {selected: false, expanded: false};
+  }
+
+  renderButtons(props) {
+    const {onToggle, onEdit, onDelete, dataPoint, selected} = props;
+
+
+    return (
+      <Fragment>
+        <Row noGutters={true}>
+          {typeof onEdit !== "undefined" && (
+            <Col>
+              <Button
+                onClick={() => {
+                  return onEdit(dataPoint)
+                }}
+                color='primary'
+                block
+              >
+                Edit
+              </Button>
+            </Col>
+          )}
+          {typeof onDelete !== "undefined" && (
+            <Col>
+              <Button
+                onClick={() => {
+                  return onDelete(dataPoint)
+                }}
+                color='danger'
+                block
+              >
+                Delete
+              </Button>
+            </Col>
+          )}
+          {typeof onToggle !== "undefined" && (
```

Comment:
For stuff like this, you can just do `onToggle && ...` undefined is a falsey value.

Reply:
Fixed.